rustler_tests/native/rustler_test/src/test_resource.rs

```diff
@@ -109,7 +109,7 @@ pub fn resource_make_binaries(
     )
 }
 
-#[rustler::nif]
+// #[rustler::nif]
```

Comment:
Is there no test for this?

Reply:
Nope. I'll try to trace back why. Maybe this was superceded by a different NIF. With the new logic (and this will need to go in the docs), one has to `cfg`-disable unused NIFs.

Reply:
I see, I introduced it here: https://github.com/rusterlium/rustler/pull/487/files

Looks like I merged the to-be-tested behaviour into the single `resource_make_binaries` nif.